data/json/items/tool/science.json

```diff
@@ -342,6 +342,22 @@
     "material": [ "steel" ],
     "qualities": [ [ "EXTRACT", 1 ] ]
   },
+  {
+    "id": "catalytic_cracking_reactor",
+    "type": "TOOL",
+    "name": { "str": "makeshift essential oil extractor" },
+    "description": "A makeshift extraction system.  It's pretty much just a 55gal drum, a pipe to stir the solution, and a faucet at the bottom to drain the extractor.  Not the most ideal piece of equipment but it will work.",
```

Comment:
```suggestion
    "description": "A makeshift extraction system.  It's pretty much just a 55gal drum, a pipe to stir the solution, and a faucet at the bottom to drain the extractor.  Not the most ideal piece of equipment, but it will work.",
```